contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -3289,4 +3289,39 @@ boolean GetVolumePathNamesForVolumeName(String lpszVolumeName,
      *         information, call GetLastError.
      */
     boolean EnumResourceNames(HMODULE hModule, Pointer type, WinBase.EnumResNameProc proc, Pointer lParam);
+    
+    /**
+     * Retrieves information about the first module associated with a process.
+     * 
+     * @see <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms684218(v=vs.85).aspx">MSDN</a>
+     * @param hSnapshot
+     *            A handle to the snapshot returned from a previous call to the
+     *            CreateToolhelp32Snapshot function.
+     * @param lpme
+     *            A pointer to a MODULEENTRY32 structure.
+     * @return Returns TRUE if the first entry of the module list has been
+     *         copied to the buffer or FALSE otherwise.<br>
+     *         The ERROR_NO_MORE_FILES error value is returned by the
+     *         GetLastError function if no modules exist or the snapshot does
+     *         not contain module information.
+     */
+    boolean Module32First(HANDLE hSnapshot, Tlhelp32.MODULEENTRY32.ByReference lpme);
```

Comment:
I see the `com.sun.jna.FunctionMapper` interface  but I'm unsure how I could use it surgically (without potentially impacting the rest of Kernel32).

Is there something I should mark up `Module32First` and `Module32Next` with (e.g. an annotation)?

As an alternative, I posted an idea about changing MODULEENTRY32 to potentially handle both the ANSI and Unicode versions of the functions. (Incidentally, the Unicode functions do end in `-W` but the ANSI versions do *not* end in `-A` according to MSDN).